trustymail/trustymail.py

```diff
@@ -174,15 +272,19 @@ def scan(domain_name, timeout, scan_types):
     if scan_types["mx"] and domain.is_live:
         mx_scan(domain)
 
+    if scan_types["starttls"] and domain.is_live:
+        starttls_scan(domain, smtp_timeout, smtp_localhost, smtp_ports, smtp_cache)
+
     if scan_types["spf"] and domain.is_live:
         spf_scan(domain)
 
     if scan_types["dmarc"] and domain.is_live:
         dmarc_scan(domain)
 
     # If the user didn't specify any scans then run a full scan.
-    if not (scan_types["mx"] or scan_types["spf"] or scan_types["dmarc"]):
+    if not (scan_types["mx"] or scan_types["starttls"] or scan_types["spf"] or scan_types["dmarc"]):
```

Comment:
Not really related to the change, but should there also be a condition here like above for `... and domain.is_live` ?

Reply:
Good question.  @h-m-f-t?

Reply:
@IanLee1521, yeah, I think we do want that to add that! If the domain doesn't exist (isn't live), there's no reason to run any record- or service-specific scans on it.